src/views/develop/index.hbs

```diff
@@ -78,4 +78,4 @@
     <div class="row mt-2 g-3">
     </div>
   </section>
-{{/page}}
```

Comment:
Welp, the GitHub editor loves newlines. I can fix it locally if you have codestyle rules to strip the newline.

Reply:
nice catch! add NL though :)